scripts/nsls2-dev-build.sh

```diff
@@ -32,8 +32,8 @@ show_channel_urls: true
 channels:
 - $UPLOAD_CHANNEL
 - lightsource2-tag
-- conda-forge
-- defaults" > $CONDARC_PATH
+- defaults
+- conda-forge" > $CONDARC_PATH
```

Comment:
This seems a bit dangerous as we might get versions of things from conda-forge that have matching deps on conda-forge, but we will use the defaults version of the dep which may be older.